### ISSUE TITLE
Authenticating towards the cellular network

### ISSUE THREAD

**Issue by CriXson** [Biktorgj/pinephone_modem_sdk#155]
I wanted to look at how the authentication to the network of the carrier is being done in the modem, but I can't find anything related to it. The authentication is done by the chipset as I understand so is that the Hexagon QDSP6 V5 ADSP? ß

Thanks in advance for any help or information.

**Comment by Biktorgj** [comment]
Yes, network registration, SIM access etc. is all done directly in the hexagon coprocessor, and isn't handled by the userspace. All that code is a big fat binary blob sitting in /firmware/image/
